<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon()"/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1440</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 224, 169);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="header_border">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>1440</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(142, 222, 198);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="recipes">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>3</y>
      <width>769</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kristen ITC</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Recipes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="gear">
    <property name="geometry">
     <rect>
      <x>1272</x>
      <y>24</y>
      <width>43</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 224, 169);
border: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pictures/gear 1.png</normaloff>pictures/gear 1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>43</width>
      <height>43</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="plu_lg">
    <property name="geometry">
     <rect>
      <x>1265</x>
      <y>777</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(142, 222, 198);
border: 5px solid #FFFFFF;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pictures/plus-lg 1.png</normaloff>pictures/plus-lg 1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_line">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>118</y>
      <width>753</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid rgb(180, 224, 169);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>120</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pictures/search 1 (1).png</normaloff>pictures/search 1 (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="filter_btn">
    <property name="geometry">
     <rect>
      <x>1101</x>
      <y>118</y>
      <width>45</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pictures/filter 1 (1).png</normaloff>pictures/filter 1 (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="one_page_btn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>831</y>
      <width>14</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 23pt &quot;Leelawadee UI&quot;;
color:  rgb(44, 44, 44);
background-color: rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two_page_btn">
    <property name="geometry">
     <rect>
      <x>671</x>
      <y>831</y>
      <width>15</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 23pt &quot;Leelawadee UI&quot;;
color:  rgb(44, 44, 44);
background-color: rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three_page_btn">
    <property name="geometry">
     <rect>
      <x>696</x>
      <y>831</y>
      <width>15</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 23pt &quot;Leelawadee UI&quot;;
color:  rgb(44, 44, 44);
background-color: rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dots_btn">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>831</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 23pt &quot;Leelawadee UI&quot;;
color:  rgb(44, 44, 44);
background-color: rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="last_page_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>831</y>
      <width>33</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 23pt &quot;Leelawadee UI&quot;;
color:  rgb(44, 44, 44);
background-color: rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="border_recipe_one">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>235</y>
      <width>370</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 224, 169);
border: 5px solid rgb(166, 251, 210);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="border_recipe_two">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>235</y>
      <width>370</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 224, 169);
border: 5px solid rgb(166, 251, 210);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="border_recipe_three">
    <property name="geometry">
     <rect>
      <x>925</x>
      <y>235</y>
      <width>370</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 224, 169);
border: 5px solid rgb(166, 251, 210);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_one">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>340</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_two">
    <property name="geometry">
     <rect>
      <x>542</x>
      <y>256</y>
      <width>340</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_three">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>256</y>
      <width>340</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title_one">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>468</y>
      <width>318</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Leelawadee UI&quot;;
color: rgb(44, 44, 44);</string>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_one">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>522</y>
      <width>318</width>
      <height>217</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Leelawadee UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cooking process</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="title_two">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>468</y>
      <width>318</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Leelawadee UI&quot;;
color: rgb(44, 44, 44);</string>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_three">
    <property name="geometry">
     <rect>
      <x>957</x>
      <y>468</y>
      <width>318</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Leelawadee UI&quot;;
color: rgb(44, 44, 44);</string>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_two">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>522</y>
      <width>318</width>
      <height>217</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Leelawadee UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cooking process</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_three">
    <property name="geometry">
     <rect>
      <x>957</x>
      <y>522</y>
      <width>318</width>
      <height>217</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Leelawadee UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cooking process</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>90</y>
      <width>1461</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>header</zorder>
   <zorder>header_border</zorder>
   <zorder>recipes</zorder>
   <zorder>gear</zorder>
   <zorder>plu_lg</zorder>
   <zorder>search_line</zorder>
   <zorder>search_btn</zorder>
   <zorder>filter_btn</zorder>
   <zorder>one_page_btn</zorder>
   <zorder>two_page_btn</zorder>
   <zorder>three_page_btn</zorder>
   <zorder>dots_btn</zorder>
   <zorder>last_page_btn</zorder>
   <zorder>border_recipe_one</zorder>
   <zorder>border_recipe_two</zorder>
   <zorder>border_recipe_three</zorder>
   <zorder>image_one</zorder>
   <zorder>image_two</zorder>
   <zorder>image_three</zorder>
   <zorder>title_one</zorder>
   <zorder>text_one</zorder>
   <zorder>title_two</zorder>
   <zorder>title_three</zorder>
   <zorder>text_two</zorder>
   <zorder>text_three</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
